agent/README.md

```diff
@@ -150,6 +150,8 @@ echo `git rev-parse --abbrev-ref HEAD`:`git --no-pager log -n1 --format="%ct000"
     - `[POST] /partition/{partitionName}` Sets the name of the partition name that should be used for all followup 
       report dumps (see `teamscale-partition`). For reports that are not directly sent to Teamscale the generated report 
       will contain the partition name as session ID.
+    - `[GET] /message` Returns the name of the currently configured commit message.
+    - `[POST] /message/{newMessage}` Sets the commit message that should be used for all followup report dumps (see `teamscale-message`).
```

Comment:
From a REST design standpoint, this is odd. Yes, `POST` has "anything you want semantics", but having the message as part of the UIR is weird, as it is not clear that `POST /message/New` overwrites a previous `POST /message/Previous`. It's all very implicit.

If I had to design the API, I would have done `GET /message` and `PUT /message` (with the new message in the body). Likewise for `/partition`. Here, the overwriting the previous message is even backed into the HTTP specs.

Reply:
Form that perspective, I agree. But I'm still not sure if we want to change that from a compatibility standpoint.
@karottenreibe @DreierF What do you think?

Reply:
I agree from the conceptual standpoint. I guess this was done back then because it is easier to deal with in scripts etc. @goeb might have input on whether we can change that without affecting customers.